.github/workflows/testing.yml

```diff
@@ -31,18 +31,19 @@ jobs:
       with:
         python-version: ${{ matrix.python_version }}
     - name: Install dependencies
-      # TODO remove numpy version constraint once we no longer support PyTorch < 2.3
       # TODO remove triton 3.1 install if torch < 2.6 no longer supported, see #1093
+      # TODO remove triton 3.2 install if torch < 2.7 no longer supported
```

Comment:
I kind of wish that PyTorch would put an upper bound on Triton for releases builds.

Reply:
I agree, I'm not sure if this has been reported to PyTorch, I saw some issues on the triton repo, but it's not in their scope.